README.md

```diff
@@ -1,20 +1,43 @@
-**Chat:**
-
-[![Join the chat at https://gitter.im/ostis-dev/sc-machine](https://badges.gitter.im/ostis-dev/sc-machine.svg)](https://gitter.im/ostis-dev/sc-machine?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://travis-ci.com/ostis-dev/sc-machine.svg?branch=master)](https://travis-ci.com/ostis-dev/sc-machine)
 
-**Documentation**
+## Documentation
 [http://ostis-dev.github.io/sc-machine/](http://ostis-dev.github.io/sc-machine/)
 
-**Master:** 
-[![Build Status](https://travis-ci.com/ostis-dev/sc-machine.svg?branch=master)](https://travis-ci.com/ostis-dev/sc-machine)
+## Build on Ubuntu
+
+### Dependencies
+
+Install python dependencies:
+
+```sh
+pip3 install -r requirements.txt
```

Comment:
Can fail if `pip3 install setuptools wheel` hasn't been run in advance.

Reply:
Never got problems with such way. Can you share any documentation on this?

Reply:
It's likely 'cause setuptools & wheel were already installed on your system.
https://packaging.python.org/tutorials/installing-packages/#ensure-pip-setuptools-and-wheel-are-up-to-date
I've tried to run `pip3 install -r requirements.txt` on a freshly installed Linux Mint 20 and failed due to the absence of these packages.

Reply:
This is a Mint specific problem, can you update instruction in `readme` to support Mint?